### PR TITLE
[APIM 4.4.0] Fixes primary user store related issues

### PR DESCRIPTION
**Pull request by RusJaI**
Fixes https://github.com/wso2/docs-apim/issues/8439 and https://github.com/wso2/docs-apim/issues/8597

### CODE REVIEW

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 1]
```diff
@@ -1,126 +1,436 @@
...
+<tr class="header">
+<th>Configuration Name</th>
+<th>Display Name</th>
+<th>Description</th>
+</tr>
+<tr class="even">
+<td>type</td>
+<td>userstore Type</td>
+<td>This is the type of the userstore manager that we are using. For read-write LDAP userstore manager. this value
+should be read_write_ldap_unique_id.
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:24]: Shall we revisit this description?

```suggestion
<td>This is the type of the userstore manager being used. For read-write LDAP userstore manager, this value
should be `read_write_ldap_unique_id`.
```

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 2]
```diff
@@ -1,126 +1,436 @@
...
+</table>
+
+Following are the minimum userstore properties that are needed to be provided to configure Read-Write LDAP userstore manager :
+
+<table>
+<thead>
+<tr class="header">
+<th>Property Id</th>
+<th>Primary userstore Property</th>
+<th>Secondary userstore Property</th>
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:43]:
```suggestion
<th>Primary Userstore Property</th>
<th>Secondary Userstore Property</th>
```

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 3]
```diff
@@ -1,126 +1,436 @@
...
+<td>connection_password</td>
+<td>Connection Password</td>
+<td>Password for the ConnectionName user.</td>
+</tr>
+</tbody>
+</table>
+
+
+Replace the default `user_store` configuration in the `         <APIM_HOME>/repository/conf/deployment.toml        
+` file, as per your ldap server configuration. A sample configuration is given below.
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:83]:
```suggestion
Replace the default `user_store` configuration in the `<APIM_HOME>/repository/conf/deployment.toml` file as per your LDAP server configuration. A sample configuration is given below.
```

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 4]
```diff
@@ -1,126 +1,436 @@
...
-1.  To read and write to an LDAP userstore, it is important to ensure that the `ReadGroups` and `WriteGroups` properties in the `<PRODUCT_HOME>/repository/conf/deployment.toml` file are set to `true`.
+!!! note
+    In the table given below, the `Primary userstore Property` column has the `PRIMARY` userstore properties that can be configured in the `deployment.toml` file. The `Secondary userstore Property` column has the properties that can be configured for a secondary userstore through the Management Console.
+
+<table>
+<thead>
+<tr class="header">
+<th>Property Id</th>
+<th>Primary userstore Property</th>
+<th>Secondary userstore Property</th>
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:121]:
```suggestion
<th>Primary Userstore Property</th>
<th>Secondary Userstore Property</th>
```

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 5]
```diff
@@ -191,23 +501,39 @@ The configuration for the external read/write user store in the `user-mgt.xml` f
...
+The **admin** user is the super tenant that will be able to manage all
+other users, roles, and permissions in the system by using the management
+console of the product. Therefore, the user that should have admin
+permissions is required to be stored in the userstore when you start
+the system for the first time. Since the LDAP userstore can be written
+to, you have the option of creating a new admin user in the userstore
+when you start the system for the first time. Alternatively, you can
+also use a user ID that already exists in the LDAP. For information
+about the system administrator user, see [Configuring the System
+Administrator]({{base_path}}/reference/config-catalog/#super-admin-configurations) .
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:513]: Any reason to not have this in a single paragraph?

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md [hunk 6]
```diff
@@ -191,23 +501,39 @@ The configuration for the external read/write user store in the `user-mgt.xml` f
...
 
 These two alternative configurations can be done as explained below.
 
--   If the user store is read-only, find a valid user that already resides in the user store. For example, say a valid username is 'admin'. Update the `[super_admin]` section of your configuration as shown below. You do not have to update the password element as it is already set in the user store.
-    ```
+-   If you are using a user that is already in the LDAP. Find a valid user that already resides in the userstore. For
+    example, say a valid username is
+    AdminSOA.Add the following configuration to the `deployment.toml` file as shown below. You do not have to update the password element as it is already set in the userstore.
```
Comment (ashera96) [inline en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:519]:
```suggestion
-   If you are using a user that is already in the LDAP, find a valid user that already resides in the user store. For example, say a valid username is AdminSOA. Add the following configuration to the `deployment.toml` file as shown below. You do not have to update the password element as it is already set in the userstore.
```

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 1 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:24]: addressed

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 2 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:43]: addresssed

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 3 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:83]: addressed

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 4 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:121]: addressed

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 5 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:513]: fixed the formatting

File: en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md (hunk 6 above)
Comment (RusJaI) [reply en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md:519]: addressed

**Review by ashera96 — APPROVED** [review]

**Pull request merged**